.vscode/settings.json

```
@@ -1,23 +1,11 @@
 {
   // Show ruler at 100 characters to help visualize linter line-length rule
-  "editor.rulers": [
-    100
-  ],
-  // Format code every time it is saved
-  "[javascript]": {
-    // "editor.formatOnSave": true
-  },
-  "[json]": {
-    // "editor.formatOnSave": true
-  },
-  "[markdown]": {
-    // "editor.formatOnSave": true
-  },
-  "[scss]": {
-    // "editor.formatOnSave": true
-  },
-  "extensions.ignoreRecommendations": true,
-  "files.autoSave": "off",
-  "debug.allowBreakpointsEverywhere": true,
-  "gitdoc.enabled": false
+  "editor.rulers": [100],
```

```suggestion
  "editor.rulers": [80],
```

is there a specific reason we have this at 100? I just noticed this, I thought most common best practice is 80?

---

I assume it's because so much of the code was done with a 100 character limit that we'll just live with that.  (80 and 100 are both fairly commonly adopted and justifiable limits.  I don't know which is more common.)  I'd personally prefer setting it to 80 and gradually adopting that as we make changes.  But if Evan thinks that's too painful, I won't disagree.  I am interested in his reasoning, especially if I've guessed wrong.